test/src/edit/DistortableImageEditSpec.js

```diff
@@ -61,4 +61,11 @@ describe("L.DistortableImage.Edit", function() {
 
 		map.setView([41.7896,-87.6996]);
 	});
+
+	describe("#_toolbarSendDown", function () {
+		it.skip("Should set an image's toolbar z-index lower than its markers on dblclick", function() {
```

Comment:
Let's pull out a separate issue for this! What do you think?

Reply:
Oh I meant to delete this! This was an implementation I made that we decided not to go with. Thank you for spotting